Documentation/api-reference.md

```diff
@@ -192,12 +195,12 @@ Some example use cases for this API are:
 * A last-resort reset for when an ECID should no longer be used.
 
 This API is not recommended for:
-* Resetting a user's consent and privacy settings; see [Privacy and GDPR](https://aep-sdks.gitbook.io/docs/resources/privacy-and-gdpr).
+* Resetting a user's consent and privacy settings; see [Privacy and GDPR](https://developer.adobe.com/client-sdks/documentation/privacy-and-gdpr).
 * Removing existing custom identifiers; use the [`removeIdentity`](#removeidentity) API instead.
-* Removing a previously synced advertising identifier after the advertising tracking settings were changed by the user; use the [`setAdvertisingIdentifier`](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/identity/identity-api-reference#setadvertisingidentifier) API instead.
+* Removing a previously synced advertising identifier after the advertising tracking settings were changed by the user; use the [`setAdvertisingIdentifier`](#setadvertisingidentifier) API instead.
 
 > **Warning**
->The Identity for Edge Network extension does not read the Mobile SDK's privacy status, and therefore setting the SDK's privacy status to opt-out will not automatically clear the identities from the Identity for Edge Network extension. See [`MobileCore.resetIdentities`](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#resetidentities) for more details.
+>The Identity for Edge Network extension does not read the Mobile SDK's privacy status, and therefore setting the SDK's privacy status to opt-out will not automatically clear the identities from the Identity for Edge Network extension. See [`MobileCore.resetIdentities`](https://github.com/adobe/aepsdk-core-android/blob/staging/Documentation/Usage/MobileCore.md) for more details.
```

Comment:
Let's avoid full path and just provide the core repo url. 

@emdobrin @kevinlind @cacheung @timkim  I see URLs to faqs and core repos here, do you think we could have section in getting started and reference that section wherever we need links in our doc? Or should we move FAQs to repo docs as well?

| Section with links |
| --- |
| [Privacy/GDPR](https://developer.adobe.com/client-sdks/documentation/privacy-and-gdpr) |
| [Core API Ref](ttps://github.com/adobe/aepsdk-core-android/blob/staging/Documentation/Usage/MobileCore.md) |
| [EdgeIdentity FAQ](https://developer.adobe.com/client-sdks/documentation/identity-for-edge-network/faq) |

Reply:
I vote for keeping 
https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/Usage/MobileCore.md at least for [`MobileCore.resetIdentities`]. (I changed the branch to main now) If just going to the repo, user isn't going to find it right away.  We can avoid the deep link though.  This page isn't that big, user can scroll down and find it.

We can find a balance depends on the situation.  For Privacy and GDPR, I don't think we should move it to this repo, so it is better to point it to the developer.com.

EdgeIdentity FAQ, good point, we should move it to this repo as well.  I will copy that over.

I don't think we need to have a table with links.

Reply:
I am of the believer of having links to each and every section. I was just trying to see if we have a way to simplify it as per discussion in the team guild meeting. But yes I do believe we should have all the API reference directly without any redirects. And even if it is a task to update, developers should be able to move across documentation without any friction.